pkg/webhook/mutation.go

```diff
@@ -63,6 +64,7 @@ func AddMutatingWebhook(mgr manager.Manager, deps Dependencies) error {
 	eventBroadcaster := record.NewBroadcaster()
 	kubeClient := kubernetes.NewForConfigOrDie(mgr.GetConfig())
 
+	logger := log
```

Comment:
Let's just have the one variable name

Reply:
note that we grab onto the `log` struct before creating the handlers to pass to the injectors below